tests/trainer/optimization/test_manual_optimization.py

```diff
@@ -80,10 +80,6 @@ def test_multiple_optimizers_manual_no_return(tmpdir, kwargs):
     apex_optimizer_steps = []
 
     class TestModel(ManualOptModel):
-        def training_step(self, batch, batch_idx):
-            # avoid returning a value
-            super().training_step(batch, batch_idx)
```

Comment:
the comment says it!

Reply:
hmm, could you be pls more specific, how does this super call prevent it? :]